src/test/java/org/jabref/logic/xmp/XmpUtilReaderTest.java

```diff
@@ -109,4 +112,16 @@ void testReadPDMetadata() throws IOException, URISyntaxException, ParseException
 
         assertEquals(entryFromBibFile.get(), entries.get(0));
     }
+
+    /**
+     * Tests an pdf file with metadata which has no description section.
+     */
+    @Test
+    void testGetXmpMetadataWithNoDescription() throws IOException, URISyntaxException, NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+        Method method = XmpUtilReader.class.getDeclaredMethod("getXmpMetadata", PDDocument.class);
```

Comment:
You don't need reflection, the method getMetadata is implicitly caleld, you you can simple do this two lines:
```
   List<BibEntry> entries = XmpUtilReader.readXmp(Path.of(XmpUtilShared.class.getResource("no_description_metadata.pdf").toURI()), xmpPreferences);
        assertEquals(Collections.emptyList(), entries);
```

Reply:
Got it. Thanks.